1/test/s2i/configuration/config.xml

```diff
@@ -48,9 +47,9 @@
       <properties class="hudson.model.View$PropertyList"/>
     </hudson.model.AllView>
   </views>
-  <primaryView>All</primaryView>
+  <primaryView>s2i-test-config</primaryView>
   <slaveAgentPort>49187</slaveAgentPort>
-  <label>master</label>
+  <label>s2i-test-config</label>
```

Comment:
@gabemontero needed to move this here because something (probably the sync plugin?) is rewriting the config which is stripping out the comment we were using to confirm the config file was ours.

just fyi, i don't think it's a problem.